pkg/runtime/k0s/registry.go

```diff
@@ -0,0 +1,124 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k0s
+
+import (
+	"fmt"
+	"net"
+	"path/filepath"
+
+	"github.com/sealerio/sealer/common"
+	"github.com/sealerio/sealer/pkg/cert"
+)
+
+const (
+	SeaHub                      = "sea.hub"
+	RemoteAddEtcHosts           = "cat /etc/hosts |grep '%s' || echo '%s' >> /etc/hosts"
+	ContainerdLoginCommand      = "nerdctl login -u %s -p %s %s"
```

Comment:
Is this a general command? I think its readability fails to be better than writing in the code.

Reply:
This will use by join phase, so i define it as a const.